components/issue-list-item.tsx

```diff
@@ -37,10 +38,18 @@ export default function IssueListItem({issue = null}:{issue?: IssueData}) {
             case "disputed":{
                 return "danger"
             }
+            // REVIEW: redeem not exist in figma
+            case "redeemed":{
+                return "blue"
+            }
             default: {
                 return "blue"
             }
         }
+    }
+
+   function replaceRedeem(issueState: IssueState): IssueState{
```

Comment:
This should be happening on the backend, like we spoke on the daily

Reply:
Finish